packages/ui/src/components/DatePicker/PreviousDateRangePicker.tsx

```diff
@@ -42,6 +42,16 @@ const isCustomSelected = ({
 	return !foundPreset
 }
 
+function toDateTimeString(date: Date) {
+	return date.toLocaleDateString('en-us', {
+		year: 'numeric',
```

Comment:
We don't have moment in our ui library so we have to use the javascript built-ins.

Reply:
Should we add something to make this simpler? What about https://day.js.org/?

Reply:
day.js seems like a good alternative. Let's see if we add more date utilities in the future. I think it's wise to keep the component library as light as possible.